src/wallet/wallet.h

```diff
@@ -738,6 +738,7 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
      * @return next transaction order id
      */
     int64_t IncOrderPosNext(CWalletDB *pwalletdb = NULL);
+    DBErrors ReorderTransactions();
```

Comment:
Would be kind of nice to not expose DBErrors further from wallet, though I would still call this a strict improvement.